src/perception/PoseEstimatorModule.cpp

```diff
@@ -45,14 +47,14 @@ bool RcnnPoseModule::detectObjects(
   // Making sure the Message from RcnnPose is non empty
   if (marker_message == nullptr)
   {
-    dtwarn << "[RcnnPoseModule::detectObjects] nullptr Marker Message "
+    dtwarn << "[PoseEstimatorModule::detectObjects] nullptr Marker Message "
            << mMarkerTopic << std::endl;
     return false;
   }
 
   if (marker_message->markers.size() == 0)
```

Comment:
Nit: Using `empty()` is more preferred way to check its emptiness rather than `size()`.